lib/logmsg/logmsg-serialize-fixup.c

```diff
@@ -28,6 +28,56 @@
 
 #include <stdlib.h>
 
+enum
```

Comment:
I would really like to have a comment or a commit message that explains this a little more.

My understanding that the `LEGACY_DBPARSER_TYPE_` captures the current state of the `RPT_`. Meaning if `RPT_IP` is represented as 8, the `LEGACY_DBPARSER_TYPE_IP` should as well. The plan is to not change the `LEGACY_DBPARSER_TYPE_` ever, while the `RPT_` can be changed (order/extend). Is that correct, or I completely misunderstood this code?

Reply:
We capture these values once and db-parser() would never produce them again, at least as type values. 

If you look at the db-parser() changes in the same PR, it will start to emit a  LM_VT_XXX instead of RPT_*. This way RPT_* would simply capture the index of the parser, not even sure if that's being used anywhere.

.... looked at it, parser_type is only ever used in debug messages. So once this is merged the coupling between types and the RPT_* enums is discontinued.

Reply:
I have added a comment to clarify this.